packages/middleware/rollup.config.js

```diff
@@ -1,4 +1,4 @@
-import pkg from './package.json';
+import pkg from './_package.json';
```

Comment:
Why? Isn't there an option called `deprecated`? Also, we could set it to `private: true`, I mean inside the package.json, so it won't be possible to publish it anymore